plugin/trino-mysql/src/test/java/io/trino/plugin/mysql/TestMySqlTypeMapping.java

```diff
@@ -605,88 +594,75 @@ public void testTimestampFromMySql(ZoneId sessionZone)
                 // before epoch (MySQL's timestamp type doesn't support values <= epoch)
                 //.addRoundTrip("timestamp(3)", "TIMESTAMP '1958-01-01 13:18:03.123'", createTimestampType(3), "TIMESTAMP '1958-01-01 13:18:03.000'")
```

Comment:
is this comment outdated? 
i see test values like `1969-12-31 23:59:59.1230001`, which is before epoch

Reply:
Mysql `timestamp` type is not supporting negative epoch

Reply:
i understand now:

- we're testing negative epoch when we insert with MySQL datetime, eg
  ```
  // negative epoch
  .addRoundTrip("datetime(6)", "TIMESTAMP '1969-12-31 23:59:59.999995'", createTimestampType(6), "TIMESTAMP '1969-12-31 23:59:59.999995'")
  ```
- we cannot test roundtrip when inserting with MySQL timestamp, because it doesn't support negative epoch
- we should be able to test roundtrip when CT/CTAS with Trino, because Trino timestamp is mapped to datetime 


 is there a  test for last bullet?

Reply:
> is there a test for last bullet?

Yes we have it in `testTimestampCoercion`  L646 and L683:L686